File: C/1733.c
#include <stdio.h>

int main(){
	char word[100];
	scanf("%s", word);
	
	if(word[0]=='I' && word[1]=='O' && word[2]=='I' && word[3]=='\0')
		printf("IOI is the International Olympiad in Informatics.");
	else
		printf("I don't care.");
	
	return 0;
}

File: Java/1733.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
    	Scanner sc = new Scanner(System.in);
    	
    	String word = sc.nextLine();
    	
    	if(word.equals("IOI"))
    		System.out.println("IOI is the International Olympiad in Informatics.");
    	else
    		System.out.println("I don't care.");
    }
}